### ISSUE TITLE
Can't see the changes reflected on sass file

### ISSUE THREAD

**Issue by rajat-mehra05** [jatiinyadav/Books-Library-using-ReactJs#36]
- I am not able to see my changes that i have done in `.scss` file. Help me here

**Comment by SankalpaFernando** [comment]
@rajat-mehra05 use Live Sass Compiler if you're using vscode and enable the live sass compiling. 
[Live Sass Compiler](https://marketplace.visualstudio.com/items?itemName=ritwickdey.live-sass)

**Comment by jatiinyadav** [comment]
Thank you @SankalpaFernando for the support.
@rajat-mehra05 If still, you face any problems let me know.

**Comment by rajat-mehra05** [comment]
> @rajat-mehra05 use Live Sass Compiler if you're using vscode and enable the live sass compiling. [Live Sass Compiler](https://marketplace.visualstudio.com/items?itemName=ritwickdey.live-sass)

Yes I have already enabled live sass compiler and then clicked on `Watch Sass` and started editing `.scss` file. 
Still the changes aren't getting reflected on the web app. 

Is there something else I have to do ?

**Comment by jatiinyadav** [comment]
@rajat-mehra05 You can check this: https://stackoverflow.com/questions/61316604/live-sass-compiler-in-vscode-is-not-working
